website/content/api-docs/secret/aws.mdx

```diff
@@ -302,6 +304,10 @@ updated with the new attributes.
   is `iam_user`. If not specified, then no permissions boundary policy will be
   attached.
 
+- `mfa_serial_number` `(string)` - The ARN or hardware device number of the device configured
+  to the IAM user for multi-factor authentication. This is optional based on if the IAM user
+  has an MFA device set up on it in AWS.
```

Comment:
```suggestion
- `mfa_serial_number` `(string)` - The ARN or hardware device number of the device configured
  to the IAM user for multi-factor authentication. Only required if IAM user has an MFA device 
  set up in AWS.
```

Style correction: avoid "this" as a pronoun, make conditions for something being optional/required explict